src/helpers/safari-extension.ts

```diff
@@ -0,0 +1,135 @@
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
We have `randomUuid` in helpers that you can use so we don't need to add a hard dependency.

Reply:
Done!